### ISSUE TITLE
Is this package dead?

### ISSUE THREAD

**Issue by erennyuksell** [joltup/rn-fetch-blob#853]
Is this package dead?

**Comment by rililive** [comment]
Seems like it!

I found this alternative that is a fork of this repo. It seems promising since it support latest versions of react-native.

https://github.com/RonRadtke/react-native-blob-util

**Comment by erennyuksell** [comment]
@rililive thank you

**Comment by laurent22** [comment]
Yes this project is dead. I don't understand why they don't mark it as archived instead of letting people waste time creating pull requests that will never be merged.

**Comment by ikkyu-3** [comment]
This package is currently not being maintained. I found out through the [comments](https://github.com/joltup/rn-fetch-blob/pull/701#issuecomment-989331989) here.

**Comment by laurent22** [comment]
I am suggesting archiving this repository and linking to the new fork in this issue: https://github.com/joltup/rn-fetch-blob/issues/863
